src/main/resources/static/admin/AdminPanelPage.html

```diff
@@ -0,0 +1,369 @@
+                                                                                                src="adminimg/icn_trash.png"
+                                                                                                title="Trash" alt="">
+                        </td>
+                    </tr>
+                    <tr>
+                        <td><input type="checkbox"></td>
+                        <td>Dolor Lorem Amet</td>
+                        <td>Dobby</td>
+                        <td>26th May 2017</td>
+                        <td><input type="image" src="adminimg/icn_edit.png" title="Edit"><input type="image"
+                                                                                                src="adminimg/icn_trash.png"
+                                                                                                title="Trash" alt="">
+                        </td>
+                    </tr>
+                    </tbody>
+                </table>
+            </div>
+        </div>
+    </article>
+
+    <article class="module width_quarter">
+        <header><h3>Messages</h3></header>
+        <div class="message_list">
+            <div class="module_content">
+                <div class="message"><p>Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor.</p>
+                    <p><strong>Max</strong></p></div>
+                <div class="message"><p>Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor.</p>
+                    <p><strong>Max</strong></p></div>
+                <div class="message"><p>Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor.</p>
+                    <p><strong>Max</strong></p></div>
+                <div class="message"><p>Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor.</p>
+                    <p><strong>Max</strong></p></div>
+                <div class="message"><p>Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor.</p>
+                    <p><strong>Max</strong></p></div>
+            </div>
+        </div>
+        <footer>
+            <form class="post_message">
+                <input type="text" value="Message"
+                       onfocus="if(!this._haschanged){this.value=''};this._haschanged=true;">
+                <input type="submit" class="btn_post_message" value=""/>
+            </form>
+        </footer>
+    </article>
+
+    <div class="clear"></div>
+
+    <article class="module width_full">
+        <header><h3>Post New Article</h3></header>
+        <div class="module_content">
+            <fieldset>
+                <label>Post Title</label>
+                <input type="text">
+            </fieldset>
+            <fieldset>
+                <label>Content</label>
+                <textarea rows="12"></textarea>
+            </fieldset>
+            <fieldset style="width:48%; float:left; margin-right: 3%;">
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add a <legend> tag to this fieldset. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AFieldsetWithoutLegendCheck)